config/locales/client.en.yml

```
@@ -1040,7 +1040,7 @@ en:
     user_fields:
       none: "(select an option)"
       required: 'Please enter a value for "%{name}"'
-      same_as_password: 'Your password should not be repeated in other fields.'
+      same_as_password: "Your password should not be repeated in other fields."
```

Looks like an overly eager linter?

```suggestion
      same_as_password: 'Your password should not be repeated in other fields.'
```